templates/modern-solutions/rop-demo/statement.rst

```diff
@@ -0,0 +1,3 @@
+An example of using ROP to successfully call ``checker(0x11223344)``.
+
+Read and understand the contents of the ``exploit.py`` pwntools exploit.
```

Comment:
I would also advise to run `exploit.py` and I would tell them how they should run it.

Reply:
Right. Otherwise they might just read it without running it.